server/liveblog/bandwidth/tasks.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import requests
+from datetime import timedelta
+
+from superdesk.celery_app import celery
+from superdesk import get_resource_service
+from superdesk.utc import utcnow
+from settings import (
+    CLOUDFLARE_URL,
+    CLOUDFLARE_AUTH,
+    CLOUDFLARE_ZONE_TAG,
+    SUBSCRIPTION_LEVEL,
+    SUBSCRIPTION_LEVEL_GO,
+)
+
+logger = logging.getLogger("liveblog")
+
+
+@celery.task
+def fetch_bandwidth_usage():
+    """
+    Fetch bandwidth usage from Cloudflare API
+    """
+    logger.info("Fetching bandwidth usage from Cloudflare API")
+
+    if SUBSCRIPTION_LEVEL != SUBSCRIPTION_LEVEL_GO:
```

Comment:
can we instead check the instance settings like "has bandwidth limit" instead? that way it wouldn't be hardcoded only to 'GO'